includes/admin/discounts/add-discount.php

```diff
@@ -57,8 +57,8 @@
 				</th>
 				<td>
 					<input type="text" id="edd-amount" name="amount" value="" style="width: 40px;"/>
-					<p class="description edd-amount-description" style="display:none;"><?php printf( __( 'Enter the discount amount in %s', 'easy-digital-downloads' ), edd_get_currency() ); ?></p>
-					<p class="description edd-amount-description"><?php _e( 'Enter the discount percentage. 10 = 10%', 'easy-digital-downloads' ); ?></p>
+					<p class="description edd-amount-description flat" style="display:none;"><?php printf( __( 'Enter the discount amount in %s', 'easy-digital-downloads' ), edd_get_currency() ); ?></p>
```

Comment:
Just thinking that flat might not be the best idea for a class name given some plugins might globally load css, and use flat for like flat design or something like that. Maybe `discount-type-flat` would be better?